reactor-pool/src/test/java/reactor/pool/CommonPoolTest.java

```diff
@@ -94,7 +94,7 @@ public <T, CONF extends PoolConfig<T>> AbstractPool<T> apply(PoolBuilder<T, CONF
 			return (AbstractPool<T>) builder.idleResourceReuseOrder(isLru).buildPool();
 		}
 	}
-	
+
```

Comment:
nit: can you remove the extra spaces in this line ?

Reply:
> nit: can you remove the extra spaces in this line ?

It was automatically removed by my IDE. Do you mean restore extra spaces just to remove this diff? I will do it using GitHub editor (I don't want to disable this useful feature in my IDE).

Reply:
my bad, you are right, I was meaning restore the extra space, in order to keep the patch as minimal.